libs/blocks/sticky-header/sticky-header.js

```diff
@@ -0,0 +1,24 @@
+import { createTag, getConfig } from '../../utils/utils.js';
+
+export default function decorate($block) {
```

Comment:
nit: I don't love having a dollar behind of most of your variables. I can see you're using it to identify html elements, but I think it's unnecessary tbh. It's not a standard that we have and just makes all your variable names longer.